doc/source/create-k8s-cluster.rst

```diff
@@ -163,6 +169,121 @@ Setting up Kubernetes on Microsoft Azure Container Service (ACS)
 
    The response should list three running nodes.
 
+.. _amazon-aws:
+
+Setting up Kubernetes on Amazon Web Services (AWS)
+--------------------------------------------------
+
+AWS does not have native support for Kubernetes, however there are
+many organizations that have put together their own solutions and
+guides for setting up Kubernetes on AWS. We like the `Heptio guide`_,
+and recommend using this for setting up your stack.
+
+.. note::
+
+  See the `Heptio's note on security <http://blog.kubernetes.io/2016/08/security-best-practices-kubernetes-deployment.html>`_
```

Comment:
The link should be to the Heptio guide's statement about their particular tutorial build (which without modification is not designed as a production build): http://docs.heptio.com/content/tutorials/aws-cloudformation-k8s.html

From Heptio doc:
> We’ll suggest sane defaults for both Kubernetes and AWS. However, keep in mind that Kubernetes is a fast-moving project, and this stack is appropriate for proof of concept (PoC), experimentation, development, and small internal-facing projects. Consider this a test drive.

@arokemI believe @choldgraf used the link to share best practices of k8. The link (with a name change) should remain along with the one above and the text snippet in this message.